### DISCUSSION TITLE
Make Log volume panel collapsible

### DISCUSSION THREAD

**Discussion by Mario-Hofstaetter** [grafana/grafana#48160] [Dashboards, Panels & Visualizations]
As per slack discussion here: https://grafana.slack.com/archives/C0Y4TLW74/p1650487806420699

Currenty in Grafana Explore, the Log volume panel is always displayed and takes up quite a bit of screen real estate.

**Please make the Log volume panel collapsible / concealable so the user may choose when to use it.**

I find myself a lot scrolling up and down from query editor to viewing logs.
For example, on 1920x1080 [on here](https://play.grafana.org/explore?orgId=1&left=%7B%22datasource%22:%22Loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bstatus_code%3D%5C%22401%5C%22%7D%22,%22hide%22:true%7D,%7B%22refId%22:%22B%22,%22expr%22:%22%7Bstatus_code%3D%5C%22403%5C%22%7D%22,%22hide%22:false%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D) the log volume takes about 207px of 995px of visible height,

**thats more than 20 percent of the usable height.**

If Log volume makes it into dashboard panels (#46600), please make it also collapsible there.

The Log volume panel could also be space optimized a little bit, if the legend was on the right hand side, but maybe less convenient so.

Collapsable draft, courtesy of Kristina Durivage (could not find her gh handle):

![grafik](https://user-images.githubusercontent.com/33002073/165026862-cf8d8859-9f2f-473c-aec1-5489c7ac3dcb.png)

![grafik](https://user-images.githubusercontent.com/33002073/165028116-e8172518-4078-4784-9bb2-00c6a6481906.png)

Also please make the panel margins the same space, preferably the smaller one :-)

![grafik](https://user-images.githubusercontent.com/33002073/165029211-6c2196b2-db98-4eb5-bdad-aede0db4aac0.png)

**Comment by ivanahuckova** [comment]
Logs volume is now collapsible 🎉, so I am closing this discussion.
 
<img width="1905" alt="image" src="https://user-images.githubusercontent.com/30407135/230416119-45cbcc08-f64c-4fa6-b857-898ac6452fcb.png">
